spot-ml/ml_test.sh

```diff
@@ -21,6 +21,7 @@
 
 DSOURCE=$1
 RAWDATA_PATH=$2
+LDAOPTIMIZER="online"
```

Comment:
It shouldn't and also is referencing to an out of date variable name. It will read from spot.conf.